packages/@aws-cdk/aws-codecommit/package.json

```diff
@@ -98,7 +99,8 @@
     "@aws-cdk/aws-events": "0.0.0",
     "@aws-cdk/aws-iam": "0.0.0",
     "@aws-cdk/core": "0.0.0",
-    "constructs": "^3.3.69"
+    "constructs": "^3.3.69",
+    "@aws-cdk/aws-codestarnotifications": "0.0.0"
```

Comment:
Please put this at the beginning of the list, like you did for `"dependencies"`.